src/App.tsx

```diff
@@ -153,46 +158,95 @@ const App = () => {
               {errorMessage ? <div>Error: {errorMessage}</div> : null}
             </SettingsWrapper>
           </div>
-          <div className="flex grow justify-end">
+          <div className='flex grow justify-end'>
             <button
-              className="btn btn-blue self-end"
+              className='btn btn-blue self-end'
               onClick={() => setData(convertFromString(explanationData))}
             >
               Update View
             </button>
           </div>
         </SettingsWrapper>
       </div>
-      <div className="resize box-border overflow-auto border border-gray-300 max-w-screen m-auto bg-blue-50 dark:bg-blue-900">
+      <div className='flex flex-row justify-center gap-2'>
+        <SettingsWrapper title='Visualization'>
+          <div className='flex flex-col'>
+            <label className='mb-1' htmlFor='minFont'>
+              Minimum Font Size
+            </label>
+            <input
+              className='bg-transparent rounded border border-gray-300'
+              id='data'
+              name='data'
+              type='number'
+              step={1}
+              value={minFontSize}
+              onChange={(event) => {
+                const v = parseInt(event.target.value);
+                setMinFontSize(v);
+                if (v > maxFontSize) {
+                  setMinFontSize(maxFontSize);
+                } else {
+                  setMinFontSize(v);
+                }
+              }}
+            />
+          </div>
```

Comment:
You could extract this to a component so you could re-use it for the min and max.